kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -330,71 +363,48 @@ define([
-                        placeholder: 'Select a type',
-                        containerCssClass: 'kb-staging-table-body__import-dropdown'
-                    }).on('select2:select', function() {
-                        $('td:eq(5)', row).find('.select2-selection').addClass('kb-staging-table-body__import-type-selected');
-                        //make checkbox for that row enabled
-                        $('td:eq(0)', row).find('.kb-staging-table-body__checkbox-input').prop('disabled', false);
-                    });
+                    function changeImportButton(event) {
+                        const checked = event.currentTarget.checked;
 
-                    $('td:eq(5)', row).find('button[data-import]').off('click').on('click', e => {
-                        var importType = $(e.currentTarget).prevAll('select').val();
-                        var importFile = getFileFromName($(e.currentTarget).data().import);
-                        this.initImportApp(importType, importFile);
-                        this.updateView();
-                    });
-
-                    $('td:eq(5)', row).find('button[data-download]').off('click').on('click', e => {
-                        let file = $(e.currentTarget).data('download');
-                        if (this.subpath) {
-                            file = this.subpath + '/' + file;
+                        if (checked) {
+                            stagingAreaViewer.enableImportButton();
+                        } else {
+                            /* 
+                                check state of all checkboxes
+                                if any are checked we leave import button enabled
+                            */
+                            let anyCheckedBoxes = $('input.kb-staging-table-body__checkbox-input:checked');
+
+                            if(!anyCheckedBoxes.length) {
+                                stagingAreaViewer.disableImportButton();
+                            }
                         }
-                        const url = Config.url('staging_api_url') + '/download/' + file;
-                        this.downloadFile(url);
-                    });
+                    }
 
-                    $('td:eq(5)', row).find('button[data-delete]').off('click').on('click', e => {
-                        var file = $(e.currentTarget).data('delete');
-                        if (window.confirm('Really delete ' + file + '?')) {
-                            this.stagingServiceClient.delete({
-                                path: this.subpath + '/' + file
-                            }).then(() => {
-                                this.updateView();
-                            }).fail(xhr => {
-                                alert('Error ' + xhr.status + '\r' + xhr.responseText);
-                            });
-                        }
+                    $('td:eq(0)', row).find('input.kb-staging-table-body__checkbox-input')
```

Comment:
added functionality here so when checkboxes are checked we can enable or disable the import button as needed